### DISCUSSION TITLE
Get game activity for current account

### DISCUSSION THREAD

**Discussion by alvkn** [SteamRE/SteamKit#998] [General]
Can i get game activity (i need in-game hours) for current user using steamkit2? If I can, how can i do it?

**Comment by yaakov-h** [comment]
What do you mean by game activity? Can you share a screenshot of the thing you are after as it appears in Steam or in a particular game?

**Reply by alvkn** [reply]
I only need the amount of time that the current client has played in a certain game. In the screenshot it is 1655 hours. I just need this number.
![изображение](https://user-images.githubusercontent.com/78173939/121831963-87974100-ccd1-11eb-8c71-f9d89895ecf6.png)

**Reply by yaakov-h** [reply]
I don't think that information is available in any CM messages.

It does appear to be available via the [GetOwnedGames Web API](https://developer.valvesoftware.com/wiki/Steam_Web_API#GetOwnedGames_.28v0001.29) (in minutes rather than hours).

**Reply by alvkn** [reply]
this is exactly what i need, thank you
